CMakeLists.txt

```
@@ -141,6 +141,10 @@ find_package(PkgConfig REQUIRED)
 
 find_package(OpenCV REQUIRED)
 
+if(NOT (${OpenCV_VERSION} VERSION_LESS 3))
```

what do you think about that?
```suggestion
if${OpenCV_VERSION} VERSION_GREATER_EQUAL 3)
```